data/cassandra.go

```diff
@@ -43,12 +45,19 @@ func (f Filter) query() string {
 	return queryString
 }
 
-func NewCassandraConnector(path, port, keyspace string) *CassandraConnector {
+func NewCassandraConnector(path, port, keyspace, username,
+	password string) *CassandraConnector {
 	logger := util.LogInit()
-	logger.Printf("New Connector Path: %s, Port: %s, Keyspace: %s", path, port, keyspace)
+	logger.Printf("New Connector Path: %s, Port: %s, Keyspace: %s, "+
```

Comment:
Lets not log the PW here.  Go ahead an add a debug log with the pw in the short term